### PR TITLE
fix Startup failed while Assembly.GetCallingAssembly not supported

### LINKED ISSUE #218: Can't startup on UWP release mode
Windows version: Windows 10 Home 1903
OS version: 18362.535

I create a demo here:
https://github.com/h82258652/ExceptionlessUwpBugDemo

While I call ```Startup```, I got "Operation is not supported on this platform". 
Snipaste_2020-01-09_09-34-46

Debug mode is no problem, but only the release mode package can submit to the ms store.

### PR DESCRIPTION
**Pull request by h82258652**
fix #218

### CODE REVIEW

**Comment by h82258652** [comment]
Snipaste_2020-01-12_19-56-35
Strange, I have signed the CLA. But it is still pending.

File: src/Exceptionless/Extensions/ExceptionlessConfigurationExtensions.cs [hunk 1]
```diff
@@ -258,7 +258,19 @@ public static class ExceptionlessConfigurationExtensions {
...
-                config.ReadFromAttributes(Assembly.GetEntryAssembly(), Assembly.GetCallingAssembly());
+                Assembly callingAssembly;
+                try {
+                    callingAssembly = Assembly.GetCallingAssembly();
+                }
+                catch (PlatformNotSupportedException) {
+                    callingAssembly = null;
+                }
+
+                if (callingAssembly == null) {
```
Comment (niemyjski) [inline src/Exceptionless/Extensions/ExceptionlessConfigurationExtensions.cs]: We don't need to null check this here as it happens here: https://github.com/exceptionless/Exceptionless.Net/blob/997e8f0b3716acab9f21eee13f16a354c5b74458/src/Exceptionless/Extensions/ExceptionlessConfigurationExtensions.cs#L164 if you just want to pass it along like in line 272.

**Comment by niemyjski** [comment]
Thanks for the PR, I requested one change. That's really weird with the cla.

**Comment by h82258652** [comment]
@niemyjski Done, thanks for the review.

**Comment by niemyjski** [comment]
Thanks for the PR. I'm going to try and get some minor other fixes out and then push a release. Until then please feel free to use our nightly builds.

**Pull request merged**